src/howtos/setup-external-registry.md

```diff
@@ -0,0 +1,18 @@
+## How to setup external Epinio registry
+
+Epinio allows the use of an external registry.  This can be achieved by setting the following arguments during `epinio install`:
```

Comment:
```suggestion
Epinio allows the use of an external registry for the storage of application images.  This can be achieved by setting the following arguments during `epinio install`:
```